src/test/scala/chiselTests/util/IsXSpec.scala

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.util
+
+import chisel3._
+import chisel3.stage.ChiselGeneratorAnnotation
+import chisel3.testers.BasicTester
+import chisel3.util.circt.IsX
+import circt.stage.ChiselStage
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.io.Source
+
+private class IsXBundle extends Bundle {
+  val a = UInt()
+  val b = SInt()
+}
+
+private class IsXTop extends Module {
+  val io = IO(new Bundle {
+    val w = Input(UInt(65.W))
+    val x = Input(new IsXBundle)
+    val y = Input(UInt(65.W))
+    val outw = UInt(1.W)
+    val outx = UInt(1.W)
+    val outy = UInt(1.W)
+  })
```

Comment:
super nit but mixing the `Input` and "missing Output means 'Aligned'` is making my teeth hurt a bit:

```suggestion
  val io = IO(new Bundle {
    val w = Flipped(UInt(65.W))
    val x = Flipped(new IsXBundle)
    val y = Flipped(UInt(65.W))
    val outw = UInt(1.W)
    val outx = UInt(1.W)
    val outy = UInt(1.W)
  })
```